mdpow/workflows/dihedrals.py

```diff
@@ -299,6 +289,33 @@ def get_dihedral_groups(solute, atom_indices):
     return dihedral_groups
 
 def get_paired_indices(atom_indices, bond_indices, dihedral_groups):
+    '''Combines `atom_indices` and `bond_indices` in tuples
+       to be paired with their respective `dihedral_groups` in
+       :func:`~MDAnalysis.core.groups.select_atoms` selection string format.
```

Comment:
I don't understand what `select_atoms()` has to do with this: Is this related to the selection *string* or the resulting AtomGroup?